services/galley/src/Galley/Run.hs

```diff
@@ -60,14 +64,24 @@ run o = do
   deleteQueueThread <- Async.async $ evalGalley e Internal.deleteLoop
   refreshMetricsThread <- Async.async $ evalGalley e Internal.refreshMetrics
   let rtree = compile sitemap
+      app :: Application
       app r k = runGalley e r (route rtree r k)
+      -- the servant API wraps the one defined using wai-routing
+      servantApp :: Application
+      servantApp r =
+        Servant.serve
+          -- we don't host any Servant endpoints yet, but will add some for the
+          -- federation API, replacing the empty API.
+          (Proxy @(Servant.EmptyAPI :<|> Servant.Raw))
```

Comment:
Why not this?

```suggestion
          (Proxy @Servant.Raw)
```

I don't suggest it's better, just curious.

Reply:
This is where I will put `Federation.Api` soon, so I wanted to already put as much of the structure in place. No deeper reason.